tests/test-kubernetes.sh

```diff
@@ -30,12 +30,19 @@ kubectl_config() {
 kubectl_deploy() {
     kubeclt_clean
 
+    echo "install Istio"
+    curl -L https://git.io/getIstio | sh -
+    cd $(ls | grep istio)
```

Comment:
As a best practice, please use `pushd` and `popd` rather than `cd` and `cd ..`.

Reply:
Will change to pushd and popd in the next commit.